src/current/v24.1/cluster-settings.md

```diff
@@ -7,12 +7,17 @@ docs_area: reference.cluster_settings
 
 Cluster settings apply to all nodes of a CockroachDB cluster and control, for example, whether or not to share diagnostic details with Cockroach Labs as well as advanced options for debugging and cluster tuning.
 
-They can be updated anytime after a cluster has been started, but only by a member of the `admin` role, to which the `root` user belongs by default.
-
 {{site.data.alerts.callout_info}}
 In contrast to cluster-wide settings, node-level settings apply to a single node. They are defined by flags passed to the `cockroach start` command when starting a node and cannot be changed without stopping and restarting the node. For more details, see [Start a Node]({% link {{ page.version.version }}/cockroach-start.md %}).
 {{site.data.alerts.end}}
 
+This page provides information on:
+
+- [Public settings](#settings)
```

Comment:
```suggestion
- [Available settings](#settings)
```

Reply:
I think if you run `SHOW CLUSTER SETTINGS` one of the columns is `Public` (true/false), so maybe we should keep it as `Public`?